src/odemis/gui/comp/overlay/world.py

```diff
@@ -3049,8 +3050,30 @@ def on_left_up(self, evt):
 
     def draw(self, ctx, shift=(0, 0), scale=1.0):
         """ Draw the selection as a rectangle. Exactly the same as parent function except that
-         it has an adaptive line width (wider if the overlay is active). """
+         it has an adaptive line width (wider if the overlay is active) and it always shows the
+         size label of the selected rectangle. """
         line_width = 5 if self.active.value else 2
+
+        # show size label if ROA is selected
+        if self.active.value:
+            # Important: We need to use the physical positions, in order to draw
+            # everything at the right scale.
+            offset = self.cnvs.get_half_buffer_size()
+            b_start_pos = self.cnvs.phys_to_buffer(self.p_start_pos, offset)
+            b_end_pos = self.cnvs.phys_to_buffer(self.p_end_pos, offset)
+            b_start_pos, b_end_pos = self._normalize_rect(b_start_pos, b_end_pos)
+
+            w, h = (abs(s - e) for s, e in zip(self.p_start_pos, self.p_end_pos))
+            w = units.readable_str(w, 'm', sig=2)
+            h = units.readable_str(h, 'm', sig=2)
+            size_lbl = u"{} x {}".format(w, h)
+
+            pos = Vec(b_end_pos.x - 8, b_end_pos.y + 5)
```

Comment:
why `-8` and `+5`?

Reply:
This is code copied from the inherited class. I think it is for not showing it exactly at the top/right corner of the ROA but a little bit shifted to the left.